templates/includes/pagination.html

```diff
@@ -0,0 +1,12 @@
+{% if is_paginated %}
+  <ul class="pagination">
+    {% if page_obj.has_previous %}
+      <li class="page-item"><a class="page-link" href="?page={{ page_obj.previous_page_number }}">Prev</a></li>
+    {% endif %}
+    <li class="page-item active"><span class="page-link">{{ page_obj.number }} of {{ paginator.num_pages }}</span>
```

Comment:
The current page number and total number of pages are displayed correctly using `{{ page_obj.number }}` and `{{ paginator.num_pages }}`. This provides users with clear navigation information.